### ISSUE TITLE
Registering a module like random module in jax as default

### ISSUE THREAD

**Issue by to24toro** [Qiskit/arraylias#16]
### What is the expected behavior?
If you want to use random.seed, random.unform, and so on in jax, you have to use `from jax.random import PRNGKey`  and how to use the random module is different from the numpy one.
So I would like to implement functions for random module in jax, make it so simpler and enable us to use like numpy random when registering jax modules.
